.github/workflows/export_config.yml

```
@@ -11,7 +14,19 @@ jobs:
       image: ghcr.io/dpc-sdp/bay/ci-builder:5.x
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+      - name: Log into registry ghcr.io
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

In the other instances where a workflow needs to auth with the registry ([github-actions](https://github.com/dpc-sdp/github-actions/pull/36/files) and [bay](https://github.com/dpc-sdp/bay/blob/155ef541820dba04bec89e4975b0cedc58f5edb8/.github/workflows/build-deploy.yml#L35C14-L35C14)) I've been using the org secrets `REGISTRY_USER` and `REGISTRY_TOKEN`, that's a good thing because that token is scoped only to write packages. Bit of a pain because the secrets will need to be passed through from the caller but it's more reliable in getting the workflow to auth, otherwise if I were to trigger the workflow using a dispatch the workflow may fail on this step. May be a worth a huddle to discuss how workflows can set required scopes @FleetAdmiralButter ?

```suggestion
          username: ${{ secrets.REGISTRY_USER }}
          password: ${{ secrets.REGISTRY_TOKEN }}
```

---

Hey @GROwen, by default the `GITHUB_TOKEN` injected by Actions should have read/write permissions to packages in the repository: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token - unless the default org permissions are set to "restricted". Not sure if this is the case of the SDP org?